.github/workflows/node-ci.yml

```diff
@@ -39,16 +34,113 @@ jobs:
       - name: 📦 Install dependencies
         run: yarn --production=false --frozen-lockfile
 
-      - name: 🔨 Build
-        run: yarn build --verbose
-
       - name: 💅🏼 Lint
         run: |
           yarn lint
           yarn ci:lint-docs
 
+  node-build:
```

Comment:
I wonder if there's a way to re-use all of this duplication :/

Reply:
Doesn't seem so - https://github.com/actions/starter-workflows/issues/245. 

I stumbled across the `needs` field while looking through their [docs](https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#jobsjob_idneeds). Couldn't get it to work properly and it arguably ends up creating more duplication - https://github.com/Shopify/quilt/pull/1705/commits/89013b8e737a1e0dd7d1b0f777ddf67aca589456.

Reply:
Not yet, it seems: https://github.community/t/reusing-sharing-inheriting-steps-between-jobs-declarations/16851/15